### ISSUE TITLE
How to use StitcherService to directly play an mp3 file as a voiceover?

### ISSUE THREAD

**Issue by arman-sydikov** [ManimCommunity/manim-voiceover#80]
As you can see from the screenshot, a demonstration video [VoiceoverDemo.mp4](https://github.com/ManimCommunity/manim-voiceover?tab=readme-ov-file#manim-voiceover) explains at 2:20 that with `StitcherService` it is possible to directly play an mp3 file as a voiceover.

<img width="828" alt="Screenshot 2024-01-24 at 22 48 57" src="https://github.com/ManimCommunity/manim-voiceover/assets/10354348/92e82a0a-bfd4-40b5-b664-e557aa6439b1">

However when I try to use it a `NameError` occurs: name 'StitcherService' is not defined
```python
from manim import *
from manim_voiceover import VoiceoverScene

class AzureExample(VoiceoverScene):
    def construct(self):
        # self.set_speech_service(
        #     AzureService(
        #         voice="ru-RU-DmitryNeural",
        #         style="newscast-casual",
        #         global_speed=1.15
        #     )
        # )
        self.set_speech_service(
            StitcherService("test.mp3")
        )
```

When I tried to fix it by importing StitcherService a `ImportError` occurred: cannot import name 'StitcherService' from 'manim_voiceover.services.stitcher' (/usr/local/lib/python3.11/site-packages/manim_voiceover/services/stitcher.py) 

```python
from manim import *
from manim_voiceover import VoiceoverScene
from manim_voiceover.services.stitcher import StitcherService

class AzureExample(VoiceoverScene):
    def construct(self):
        # self.set_speech_service(
        #     AzureService(
        #         voice="ru-RU-DmitryNeural",
        #         style="newscast-casual",
        #         global_speed=1.15
        #     )
        # )
        self.set_speech_service(
            StitcherService("test.mp3")
        )
```

So how to directly play an mp3 file as a voiceover?

**Comment by mushunrek** [comment]
I have run into the same problem. At the moment, the class has been renamed to `_StitcherService`, presumably because it is being worked on (?). In particular, I have noticed a bug, see #88. If you want to use `_StitcherService` anyway, you will have to go into the source code and modify the class as described in #88. This is only a workaround and should probably not be done long term... But at the moment, it works fine for me.
